src/packet_analysis/protocol/tcp/TCPSessionAdapter.cc

```diff
@@ -389,6 +391,14 @@ static zeek::RecordValPtr build_syn_packet_val(bool is_orig, const zeek::IP_Hdr*
 			winscale = options[2];
 			break;
 
+		case 8:  // TCPOPT_TIMESTAMP
+			if (opt_len < 10 )
+				break;  // bad length
+
+			TSval = ntohl(*reinterpret_cast<const uint32_t*>(options + 2));
+			TSecr = ntohl(*reinterpret_cast<const uint32_t*>(options + 6));
```

Comment:
per above, I think this will need to read the bytes individually because the pointer may not be aligned as required.